packages/desktop-client/src/components/modals/BudgetSummary.tsx

```diff
@@ -35,13 +36,18 @@ function ToBudget({ toBudget }) {
   );
 }
 
-function BudgetSummary({ month, modalProps }) {
-  const prevMonthName = monthUtils.format(monthUtils.prevMonth(month), 'MMM');
+type BudgetSummaryProps = {
+  modalProps: CommonModalProps;
+  month: string;
+};
+
+function BudgetSummary({ month, modalProps }: BudgetSummaryProps) {
+  const prevMonthName = format(prevMonth(month), 'MMM');
 
   return (
-    <Modal title="Budget Details" {...modalProps} animate>
+    <Modal title="Budget Details" {...modalProps}>
```

Comment:
Removed animate prop as it doesn't get used AFAIK. It's nowhere else in the code - seems like an anomaly.

Verified, it was still working: 
![animates by default](https://github.com/actualbudget/actual/assets/5285928/e6aff9ac-679c-44a4-9460-d4a25b6dc2f7)